backend/api/services/CreditTradeService.py

```diff
@@ -178,14 +178,16 @@ def approve(credit_trade, update_user=None, batch_process=False):
         Sets the Credit Transfer to Approved
         """
         status_approved = CreditTradeStatus.objects.get(status="Approved")
+        effective_date = credit_trade.trade_effective_date
```

Comment:
just want to confirm that on historical data entry, is the trade_effective_date always set? or can it be null?

Reply:
I checked in the prod table and many records had null value.